### DISCUSSION TITLE
copy variant nomenclature

### DISCUSSION THREAD

**Discussion by JMdeSteAgathe** [vidboda/MobiDetails#47] [Ideas]
Hi,
it would be so nice to be able to copy the nomenclature without having to select it.
my suggestion: `Copy field` button, becoming `Field copied` once clicked
or the copy icon: <img src="https://cdn-icons-png.flaticon.com/512/60/60990.png" width="20">
😁
JM

**Comment by vidboda** [comment]
Hi JM,

noted, I'm currently working on a new release due to ~1 week or 2, this could be included. You mean all kind of nomenclatures?

**Reply by JMdeSteAgathe** [reply]
Great. thanks+++
yes, if possible. The inclusion of `(hg19)` or `(hg38)` into the chr:g. and the vcf string would be great. always annoying to add them by hand, (and error prone)
```
chr1(hg38):g.154601510C>A
hg38-1-154601510-C-A
